mlshvch/hometask-6/scrapper.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'csv'
+require 'capybara'
+require 'capybara/dsl'
+require 'selenium-webdriver'
+
+SCRAPING_WEBSITE = 'https://www.onliner.by'
+NEWS_CSS_SELECTORS = %w[widget-1-1 widget-2-1 widget-3-1 widget-4-1 widget-5-1 widget-6-1 widget-7-1 widget-8-1
+                        widget-9-1 widget-10-1 widget-11-1 widget-12-1].freeze
+CATALOG_CSS_SELECTOR = 'div.catalog-offers__item'
+PEOPLE_NEWS_SELECTOR = 'ul.b-opinions-main-2'
+# class for parsing webpage data
+class Scraper
```

Comment:
A recommendation: name your classes exactly like a file name where it is placed. Will save you a couple of gray hairs when you will start working with Rails. When I import a scra**pp**er.rb file, I expect to see a `Scrapper` class, not `Scraper`, so please make it consistent.

Reply:
Thanks for advice